source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -59,6 +59,23 @@ Insert a Document
 -----------------
 
 Use the ``InsertOne()`` function to insert a single document into a collection.
+You can modify the behavior of ``InsertOne()`` by constructing and passing
+an optional ``InsertOneOptions`` struct.
+
+The available options to tune with ``InsertOneOptions`` are:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large
+
+   * - Option
```

Comment:
Suggestion: I think it would be more helpful to provide the function the user should append.